src/views/layout.ejs

```diff
@@ -1,18 +1,23 @@
 <html>
-  <head>
-    <title>GirlsCodeMK Library app</title>
-  </head>
 
-  <body>
+<head>
+  <title>GirlsCodeMK Library app</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
```

Comment:
Can this be installed via yarn?

Reply:
What would be the advantage of installing with yarn/npm over cdn @kalan? I put it as cdn so that it's more obvious how and where it is used (i.e. that we are just working with a normal HTML page) :)

Reply:
Normally I'll eventually have some custom css/js, so I'll import the bootstrap dependency into those files. Eventually going through a preprocessor (SASS) or a minifier. Before serving it.